src/index.js

```diff
@@ -0,0 +1,66 @@
+let temp=83;
+
+const changeTempColor = (currentTemp)=> {
+    const tempValueEl = document.getElementById("tempValue");
+    let color = "teal";
+    if (currentTemp>=80) {
+        color='orange';
+    } else if (currentTemp>=70) {
+        color='yellow';
+    }else if (currentTemp>=60) {
+        color='green';
+    } else if (currentTemp>=50) {
+        color='blue';
+    }
+    tempValueEl.classList = color;
+}
+
+const updateTemp = () => {
+    const tempValueEl = document.getElementById("tempValue");
+    tempValueEl.textContent=temp;
+    changeTempColor(temp);
+}
+
+const clickIncreaseTemp=()=> {
+    const tempEl=document.getElementById('tempValue')
+    temp+=1;
+    tempEl.textContent = temp
+};
+const clickDecreaseTemp=()=> {
+    const tempEl=document.getElementById('tempValue')
+    temp-=1;
+    tempEl.textContent = temp
+};
+
+const updateCityName = () => {
+    const inputName = document.getElementById("cityNameInput").value;
+    const cityName = document.getElementById("cityName");
+    cityName.textContent = inputName;
+};
+
+const resetCityName = () => {
+    const cityNameInput = document.getElementById("cityNameInput");
+    cityNameInput.value = "Los Angeles";
+    updateCityName();
+};
+
+
+
+const registerHandlers=()=> {
```

Comment:
After you create a function to update your garden then you would want to register it like you did the others. The updateGarden function should update your landscape based on your updated temp